### PR TITLE
[TESTMERGE ME]Hornygranny Cloning Prototype

### PR DESCRIPTION
**Pull request by Iamgoofball**
Cloners now stuff a random ghost into the clone upon completion.

You can't build normal cloners anymore or upgrade the cloners for autoclone.

Q&A:
Q: Why?
A: HG suggested it and Arm agreed to testmerge.
Q: Muh snowflake character
A: Not my problem buddy, we have changelings so you can get over it
Q: I dont like not being able to guarantee it's me upon revival
A: Tough shit man, get defibbed or Strange Reagent if you want a guaranteed revive.

### CODE REVIEW

**Comment by hornygranny** [comment]
:+1:

**Comment by Supermichael777** [comment]
I only want this because i know it will cause a violent reactionary fervor and cloning will bee memeticly anchored to the way it was before the ban

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -383,15 +383,6 @@
 	export_price = 5000
 
 ////////////////////////Medical////////////////////////
-/datum/techweb_node/cloning
-	id = "cloning"
-	display_name = "Genetic Engineering"
-	description = "We have the technology to make him."
-	prereq_ids = list("biotech")
-	design_ids = list("clonecontrol", "clonepod", "clonescanner", "scan_console", "cloning_disk")
```
Comment (Supermichael777) [inline code/modules/research/techweb/all_nodes.dm]: cloning scanners are used in normal genetics BTW or at least they used to be

**Comment by optimumtact** [comment]
feature/tweak during the freeze

**Comment by PKPenguin321** [comment]
Whatever happened to the hardcore branch

**Comment by 81Denton** [comment]
>get defibbed or Strange Reagent if you want a guaranteed revive

You could gibe /tg/ docs defib arms and a strange reagent dispenser and they'd still fuck it up

**Comment by Spaceinaba** [comment]
whats the fucking point of this
also if arm actually said he would testmerge this I will kill him

**Comment by chesse20** [comment]
HUGE SUPPORT PUT THIS IN THE GAME NOW PLEASE and make it PERMAnet

**Comment by sawrge** [comment]
A very frequently used feature robbing people of their characters when this is ostensibly advertised as a rolepalying game seems really silly

This might be cool as a small % chance but I don't understand it being the default behavior

There's also been lots of historic problems of players impersonating other players' characters leading to all sorts of headache inducing shenanigans

**Comment by Iamgoofball** [comment]
we have ui+ue syringes and a whole gamemode based around impersonating player characters its fine

**Comment by PKPenguin321** [comment]
yes that is of course exactly the same

**Comment by RogueSteampunker** [comment]
"we have ui+ue syringes and a whole gamemode based around impersonating player characters its fine"

>Goof assumes people are pissed about people playing their characters
>In reality, people don't want to basically be told that even if their body is found, there's no chance of them coming back in.

Goof, you really, REALLY can just say that you don't play, or just admit that you're bored and don't want to make another dogborg removal thread on cit for the sake of attention.

**Comment by Nilons** [comment]
I don't see too much wrong with nerfing cloning in a meaningful way when there are still so many other ways to bring people back.

**Comment by Armhulen** [comment]
Your point was good until you said pod people were worse humans

**Comment by RogueSteampunker** [comment]
@Armhulen they really are, though. Silicons having no reason to regard you in a positive eye is a serious downside, and being able to be ended with a simple chemical or an emp is also really iffy. Sure, regening in light is alright, but it just doesn't add up in my eyes with the serious downsides that being a podperson bring.

**Pull request closed without merging**